src/index.tsx

```diff
@@ -3,15 +3,25 @@ import ReactDOM from 'react-dom/client';
 import "./styles.scss"
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { ApolloClient, InMemoryCache, ApolloProvider } from "@apollo/client"
```

Comment:
Please sort your imports. I wrote the order how it should be in your other PR, check it.